src/Microsoft.Identity.Web/AzureADB2COpenIDConnectEventHandlers.cs

```diff
@@ -31,7 +31,7 @@ public AzureADB2COpenIDConnectEventHandlers(
 
         public MicrosoftIdentityOptions Options { get; }
 
-        public Task OnRedirectToIdentityProvider(RedirectContext context)
+        public Task OnRedirectToIdentityProviderAsync(RedirectContext context)
```

Comment:
Is this a breaking change?

Reply:
Yes, it is. Let's not change any public API and open an issue to track changes later.

Reply:
Would the same apply to `internal` methods due to how they're shared between libraries?

Reply:
This was unintentional. The analyzers didn't catch it, and I think it got past me because the class is internal. I've added a suppression and kept the original name.